web/text_layer_builder.js

```diff
@@ -122,8 +131,8 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
                                                              matchesLength) {
       var i = 0;
       var iIndex = 0;
-      var bidiTexts = this.textContent.items;
-      var end = bidiTexts.length - 1;
+      let bidiTextsStr = this.textContentItemsStr;
```

Comment:
If you're going to change the local variable name anyway, how about using consistent names instead (applies below as well)?
I.e. `let textContentItemsStr = this.textContentItemsStr;`.